### ISSUE TITLE
False positive with two back to back properties

### ISSUE THREAD

**Issue by kjlubick** [PolymerLabs/polylint#86]
Running polylint 2.9 on the something containing the following:
```
<a href="foo/bar">{{alpha}} - {{beta}}</a>
```

yields the following error:
Property alpha}} - {{beta not found in properties for element

I think the regex is being a bit greedy on its element name.
